components/vega-visualization/README.md

```diff
@@ -0,0 +1,3 @@
+<!-- @license CC0-1.0 -->
+
+# Vega Visualization
```

Comment:
add documentation for
- how to use vega component with a spec
- how to customise styling
- accessibility